.github/workflows/codespell.yml

```diff
@@ -0,0 +1,18 @@
+# GitHub Action to automate the identification of common misspellings in text files
+# https://github.com/codespell-project/codespell
+name: codespell
+on: [push, pull_request]
+jobs:
+  codespell:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - run: pip install codespell flake8
+      - run: |
+          SKIP="./bin/runtest,./bin/trace,./bin/h5redeploy.in,./bin/snapshot,./tools/testfiles/tstring2.ddl,./hl/tools/h5watch/h5watch.c,./tools/test/h5jam/tellub.c,./tools/libtest/h5tools_test_utils.c,./src/H5T.c,./release_docs/RELEASE.txt,./config/cmake_ext_mod/NSIS.template.in,./java/src/jni/h5sImp.c,./src/H5Pdapl.c"
```

Comment:
RELEASE.txt should always be checked! That is one file that is viewed more then any other (it is always in the binaries)